sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -0,0 +1,54 @@
+# Change Log azure-storage-blob
+
+> See [BreakingChanges](BreakingChanges.md) for a detailed list of API breaks.
+
+## Version 12.0.0:
```

Comment:
12.0.0b1?

Reply:
Good point - I need to revisit the HISTORY and populate it with the release notes (and the correct version.....)

Reply:
Fixed :)